pages/common/if.md

```diff
@@ -0,0 +1,11 @@
+# if
+
+> Simple shell conditional
+
+- Perform a command depending on another command's success.
+
+`{{cmd}} && echo "success" || echo "failure"``
```

Comment:
I know. But do we want to stick to precise definition of what is a command? Because shell built-ins are not commands.
Point is, if I want to do a conditional, I will look for `if` (or `test`, as you suggested).
A `&& ||` pattern is a very good substitute for an if, isn't it?